src/Pages/Home.jsx

```diff
@@ -14,13 +15,13 @@ export default function Home() {
           variant="outlined"
           aria-label="outlined button group"
         >
-          <a href="/login">
+          <Link href="/login">
             <Button>Login</Button>
-          </a>
-          <a href="/sign-up">
+          </Link>
+          <Link href="/sign-up">
             {" "}
             <Button>Sign Up</Button>
-          </a>
+          </Link>
```

Comment:
Link comes with a `to` attribute for the url instead of `href`.

`<Link to="/about">About</Link>`